app/configurations/config.hsl.js

```diff
@@ -20,7 +20,7 @@ export default {
 
   URL: {
     OTP: process.env.OTP_URL || `${API_URL}/routing/v1/routers/hsl/`,
-    STOP_MAP: `${MAP_URL}/map/${MAP_VERSION}/hsl-stop-map/`,
+    STOP_MAP: `${process.env.OTP_URL}vectorTiles/stops,stations/`,
```

Comment:
This is very optional comment/idea.

Just thinking if, instead of string concatenation, we should build urls in such a manner that no assumption of trailing slash would be necessary. In existing codebase, there's a utility function https://github.com/HSLdevcom/digitransit-ui/blob/v2/app/util/path.js#L36 with reasonable test coverage and could be utilized as:

```javascript
import { pathJoin } = require('../util/path.js');

// ...
{
  //...
  STOP_MAP: pathJoin([process.env.OTP_URL, "vectorTiles/stops,stations"])
}
``` 

Preferably all url's should be created as  native javascript class `URL` (see: https://developer.mozilla.org/en-US/docs/Web/API/URL). For this instance, it might require unfeasible refactoring but maybe consider using `URL` in upcoming development.